setup.cfg

```diff
@@ -3,8 +3,13 @@ data_file = build/test/coverage/python_temp/.coverage
 branch = True
 omit =
     girder/external/*
-    girder/test/*
+    test/*
 parallel = True
+[coverage:paths]
+# Include sources from installed package in Tox's {envsitepackagesdir}
+girder =
+    girder/
+    build/test/tox/*/lib/*/site-packages/girder/
```

Comment:
Just to clarify the intent here - this is so that running `pytest` will always work and generate coverage for the currently checked out version of Girder, and running `tox` will always work and generate coverage against the installation of the currently checked out version of Girder.

Reply:
Yes, that's the correct intent.

The mechanism for doing so is actually the `--cov={envsitepackagesdir}/girder` argument to `pytest` in `tox.ini`. This particular config setting just ensures that Coverage's reports map all files back to the correct paths (so it will have no effect if running from the uninstalled code).

Let me know if you'd like to see additional code documentation around this, in case its non-intuitive from reading the config files.